### PR TITLE
Tableau.Public version 22.3.0

### PR DESCRIPTION
**Pull request by s-crypt**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/87104)

### CODE REVIEW

**Comment by Trenly** [comment]
```
[1F78:1F40][2022-11-01T13:14:03]e000: Error 0xffffffff: The installation failed because the installation directory is invalid.  A full path, including a drive letter, must be specified.  A UNC path, such as \\server\share, is also valid.   A path with just a drive letter, such as D:\, is not valid.
[1F78:1F40][2022-11-01T13:14:03]e000: Error 0xffffffff: Failed to initialize data in bootstrapper application.
[1F78:1F68][2022-11-01T13:14:03]i500: Shutting down, exit code: 0xffff
```

**Comment by s-crypt** [comment]
Interesting. The installer is the same as the previous version but is now having errors with the new version. 
I am not sure why the default installer path is not working, as I tested it on my machine with no problems, though there is a UAC prompt. Could that be the problem?
There is currently [no documentation](https://learn.microsoft.com/en-us/windows/package-manager/package/winget-validation#pull-request-labels) for "Validation-Shell-Execute"

File: manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Tableau.Public
+PackageVersion: 22.3.0
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml:5]:
```suggestion
PackageVersion: 22.3.1408
```

File: manifests/t/Tableau/Public/22.3.0/Tableau.Public.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Tableau.Public
+PackageVersion: 22.3.0
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Public/22.3.0/Tableau.Public.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 22.3.1408
```

File: manifests/t/Tableau/Public/22.3.0/Tableau.Public.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Tableau.Public
+PackageVersion: 22.3.0
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Public/22.3.0/Tableau.Public.yaml:5]:
```suggestion
PackageVersion: 22.3.1408
```

File: manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: ACCEPTEULA=1
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Tableau Public 2022.1 (20221.22.0331.2015)
+  Publisher: Tableau Software, LLC
+  ProductCode: '{4e57e52e-17bd-4325-8436-082178b08a83}'
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml:20]:
```suggestion
  ProductCode: '{36297416-d815-4cde-951f-dc0a58414a1b}'
```

File: manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  InstallerType: burn
+Installers:
+- Architecture: x86
+  InstallerType: burn
+  InstallerUrl: https://downloads.tableau.com/public/TableauPublicDesktop-64bit-2022-3-0.exe
+  InstallerSha256: 49A6A801B9D51F9A0A626DE787BD7F2F41FC16F48BB88DDF9F1B0A0D5431E3B2
+  ProductCode: '{4e57e52e-17bd-4325-8436-082178b08a83}'
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml:27]:
```suggestion
  ProductCode: '{36297416-d815-4cde-951f-dc0a58414a1b}'
```

File: manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: ACCEPTEULA=1
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Tableau Public 2022.1 (20221.22.0331.2015)
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml:18]:
```suggestion
- DisplayName: Tableau Public 2022.3 (20223.22.1005.1835)
```

File: manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  Publisher: Tableau Software, LLC
+  ProductCode: '{4e57e52e-17bd-4325-8436-082178b08a83}'
+  InstallerType: burn
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/t/Tableau/Public/22.3.0/Tableau.Public.installer.yaml:23]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**